.github/workflows/documentation-tests.yaml

```
@@ -34,31 +34,46 @@ env:
 jobs:
   ubuntu-build-and-test-documentation:
     name: Documentation build and test
-
-    if: ${{ !(contains(github.event.pull_request.labels.*.name, 'no-test') ||
-              contains(github.event.pull_request.labels.*.name, 'skip-ci')) }}
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
 
     runs-on: ubuntu-22.04
     steps:
       - name: Sync eProsima/Fast-DDS repository
-        uses: actions/checkout@v3
+        uses: eProsima/eProsima-CI/external/checkout@v0
         with:
           path: src/fastrtps
           submodules: true
 
-      - name: Install Fast DDS packages
-        uses: ./src/fastrtps/.github/actions/install-apt-packages
-
-      - name: Install documentation packages
-        uses: ./src/fastrtps/.github/actions/install-apt-packages-documentation
+      - name: Get minimum supported version of CMake
+        uses: eProsima/eProsima-CI/external/get-cmake@v0
+        with:
+          cmakeVersion: '3.20.6'
 
       - name: Setup CCache
         uses: eProsima/eProsima-CI/external/setup-ccache-action@v0
 
-      - name: Download FastDDS dependencies
-        run: |
-          pip3 install vcstool
-          vcs import --skip-existing src < ./src/fastrtps/fastrtps.repos
+      - name: Install apt dependencies
+        uses: eProsima/eProsima-CI/ubuntu/install_apt_packages@v0
+        with:
+          packages: curl grep wget libssl-dev doxygen imagemagick plantuml
+          update: false
+          upgrade: false
+
+      - name: Install Python dependencies
+        uses: eProsima/eProsima-CI/multiplatform/install_python_packages@v0
+        with:
+          packages: vcstool
+          upgrade: false
+
+      - name: Install colcon
+        uses: eProsima/eProsima-CI/ubuntu/install_colcon@v0
+
+      - name: Download Fast DDS .repos
+        uses: eProsima/eProsima-CI/ubuntu/vcs_import@feature/vcs_import/skip_existing
```

Please, remember to unset this before merging, once the patch release has been done in [eProsima-CI](https://github.com/eProsima/eProsima-CI)

```suggestion
        uses: eProsima/eProsima-CI/ubuntu/vcs_import@v0
```